packages/cli/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Change Log
-
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Add `--ignore-not-found` flag logic to zosfilesBase.handler and update command definitions and en.ts for delete operations. [#2254](https://github.com/zowe/zowe-cli/pull/2254)
```

Comment:
Suggesting to change this to something like: "Add the `--ignore-not-found` flag to avoid error messages potentially interrupting a script for automated batch processes. The flag bypasses warning prompts to confirm delete actions."